cms/static/sass/assets/_graphics.scss

```diff
@@ -7,7 +7,7 @@
 	width: 7px;
 	height: 22px;
 	margin-left: ($baseline/2);
-	background: url(../images/drag-handles.png) no-repeat;
+	background: url($static-path + '/images/drag-handles.png') no-repeat;
```

Comment:
Please see [my comments about the formatting for passing this path variable](https://github.com/edx/edx-platform/pull/8271/files#r37541733) into rules like this.